x-pack/plugins/secops/public/components/load_more_table/index.tsx

```diff
@@ -32,11 +32,12 @@ export interface ItemsPerRow {
 export interface SortingBasicTable {
   field: string;
   direction: Direction;
+  allowNeutralSort?: boolean;
```

Comment:
Thanks for adding this! Now we're better matching [EUI sorting](https://github.com/elastic/eui/blob/fd39fdcd272066380f1c757751ee008aef600e18/src-docs/src/views/tables/basic/props_info.js#L140).